modules/openapi-generator/src/main/resources/python/model_test.mustache

```diff
@@ -22,11 +24,20 @@ class Test{{classname}}(unittest.TestCase):
     def tearDown(self):
         pass
 
-    def test{{classname}}(self):
+    def make_instance(self, include_optional):
         """Test {{classname}}"""
-        # FIXME: construct object with mandatory attributes with example values
         # model = {{packageName}}.models.{{classFilename}}.{{classname}}()  # noqa: E501
-        pass
+        if (include_optional) :
+            return {{classname}}({{#vars}}
+                {{name}} = {{#example}}{{{.}}}{{/example}}{{^example}}None{{/example}}{{#hasMore}}, {{/hasMore}}{{/vars}})  # noqa: E501
+        else:
```

Comment:
Can you remove the else and dedent this `return blah` code here?
We don't need the else if we return in the if.

Reply:
Having the `else` makes the code clearer. Anyway, done.

Reply:
Thanks for making this change.
Can you mark this conversation as resolved?